lib/ex_aws/ses.ex

```diff
@@ -0,0 +1,187 @@
+
+defmodule ExAws.SES do
+  import ExAws.Utils, only: [camelize_key: 1, camelize_keys: 1]
+
+  @moduledoc """
+  Operations on AWS SES
+
+  http://docs.aws.amazon.com/ses/latest/APIReference/Welcome.html
+  """
+
+  @doc "Verifies an email address"
+  @spec verify_email_identity(email :: binary) :: ExAws.Operation.Query.t
+  def verify_email_identity(email) do
+    request(:verify_email_identity, %{"EmailAddress" => email})
+  end
+
+  @doc "Fetch identities verification status and token (for domains)"
+  @spec get_identity_verification_attributes([binary]) :: ExAws.Operation.Query.t
+  def get_identity_verification_attributes(identities) when is_list(identities) do
+    params = format_member_attribute({:identities, identities})
+    request(:get_identity_verification_attributes, params)
+  end
+
+  @type list_configuration_sets_opt :: {:max_items, pos_integer}
+    | {:next_token, String.t}
+
+  @doc "Fetch configuration sets associated with AWS account"
+  @spec list_configuration_sets() :: ExAws.Operation.Query.t
+  @spec list_configuration_sets(opts :: [] | [list_configuration_sets_opt]) :: ExAws.Operation.Query.t
+  def list_configuration_sets(opts \\ []) do
```

Comment:
@lessless note that this went from `configuration_sets` to `list_configuration_sets` to more closely match the AWS naming.